src/sign/Sign.service.ts

```diff
@@ -396,12 +396,12 @@ export class SignService {
       );
     }
 
-    if (!(user.roles.includes('Submitter') || user.roles.includes('Sponsor')) ) {
+    if (!(user.roles.includes('Submitter') || user.roles.includes('Sponsor') || user.roles.includes('Initial Authorizer')) ) {
       throw new EaseyException(
         new Error('This requires the Sponsor or Submitter role'),
```

Comment:
I'm thinking this is the idea:
```suggestion
        new Error('This requires the Sponsor, Submitter, or Initial Authorizer role'),
```